charts/external-dns/values.yaml

```
@@ -31,7 +31,9 @@ serviceAccount:
   create: true
   # -- Labels to add to the service account.
   labels: {}
-  # -- Annotations to add to the service account.
+  # -- Annotations to add to the service account. Templates are allowed here. Example:
+  # annotations:
+  #   example.com/annotation: {{ .Values.nameOverride }}
```

```suggestion
  # -- Annotations to add to the service account. Templates are allowed in both the key and value; for example `example.com/annotation/{{ .Values.nameOverride }}: {{ .Values.nameOverride }}`.
```
Helm docs doesn't support multi-line strings.

---

Done! thanks!